firepit/firepit.js

```diff
@@ -0,0 +1,568 @@
+// Copyright 2018, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const fs = require("fs");
+const path = require("path");
+const { fork, spawn } = require("child_process");
+const homePath = require("user-home");
+const chalk = require("chalk");
+const shell = require("shelljs");
+shell.config.silent = true;
+
+const runtime = require("./runtime");
+const version = require("./package.json").version;
+
+let config;
+try {
+  config = require("./config");
+} catch (err) {
+  console.warn("Invalid Firepit configuration, this may be a broken build.");
+  process.exit(2);
+}
+
+function SetWindowTitle(title) {
```

Comment:
If you move this function down with the other ones you can take advantage of `isWindows`